### PR TITLE
Issue 5

### PR DESCRIPTION
**Pull request by kevin-f-ortega**
Addresses https://github.com/fprime-community/fprime-workshop-led-blinker/issues/5

### CODE REVIEW

File: docs/component-implementation-1.md [hunk 1]
```diff
@@ -294,6 +294,15 @@ Finally, add the `BlinkIntervalSet` event with an argument of `U32` type to indi
 
 > Resolve any `fprime-util build` errors before continuing 
 
+## Try it yourself
```
Comment (LeStarch) [inline docs/component-implementation-1.md:297]: I had intended that the paragraph on line `293` be a "Try it yourself" table too... as that is the paragraph that is missed.  However I'd have expected multiple events in that list.

File: docs/component-implementation-2.md [hunk 2]
```diff
@@ -256,6 +256,18 @@ fprime-util build
 ```
 > Resolve any errors and finish any TODOs before continuing.
 
+## Try it yourself
```
Comment (LeStarch) [inline docs/component-implementation-2.md:259]: Same here replacing line 49.  I do like that you've called out these other things I missed....but the existing call-outs are hard to see.

**Review by LeStarch — CHANGES_REQUESTED** [review]
This is great!  Could we replace the "add new events" and "add new telemetry items" in this same format too?

I noted the lines in specific comments.  You've partially addressed this....but I think we should be tables in-place of the existing paragraphs too.

File: docs/component-implementation-1.md (hunk 1 above)
Comment (kevin-f-ortega) [reply docs/component-implementation-1.md:295]: Ah thanks, I miss that one.

File: docs/component-implementation-2.md (hunk 2 above)
Comment (kevin-f-ortega) [reply docs/component-implementation-2.md:271]: Done. I also missed it.

**Review by LeStarch — APPROVED** [review]

**Pull request merged**